index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <nav>
+        <a href="#" target="_parent" class="logo">My Portfolio</a>
+        <img src="./assets/Union.png" class="mobile_menu_icon"/>
+      </nav>
+    </header>
+
+    <section>
+      <div class="hero">
+        <h1 class="author">I’m Tanveer Glad to see you!</h1>
+        <p class="bio">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+           don’t hestiate to contact me.</p>        
+           <a href="#" class="connect_button">let's connect</a>
+
+        <div class="social">
+        <a href="#"><img src="./assets/Vector.svg" alt="twitter"></a>
+        <a href="#"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a>
+        <a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a>
+        <a href="#"><img src="./assets/Vector-1.svg" alt="github"></a>
+        <a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a>
+        </div>
```

Comment:
- [ ] I think you misunderstood the previous reviewer, the way you made this part as a `ul` and `li`s was correct 👍 but the `img` should be wrapped in an anchor tag. The final structure will look like this: 

```
<ul>
   <li> <a href="website/yourHandle"><img....></a> </li>
</ul>
```

Reply:
The purpose of these anchor tags is to take users to the given social platforms so these links do exactly as they should so you cannot say that I have to make changes and use my last review for this exactly not. I am sorry but I think I used the correct way as you can check the tons of well-designed websites where best practices were used to build them and they same have done this as I did. you can check the Microsofts website.
thank and plz approve my project that will be your most kindness.